scripts/setupTypeScriptRollup.js

```diff
@@ -0,0 +1,264 @@
+// @ts-check
+const fs = require('fs');
+const path = require('path');
+const { argv } = require('process');
+
+const projectRoot = argv[2] || path.join(__dirname, '..');
+
+console.log('Adding typescript with rollup');
+
+// Add deps to pkg.json
+const pkgJSONPath = path.join(projectRoot, 'package.json');
+const packageJSON = JSON.parse(fs.readFileSync(pkgJSONPath, 'utf8'));
+packageJSON.devDependencies = Object.assign(packageJSON.devDependencies, {
+	'@rollup/plugin-typescript': '^6.0.0',
+	'@tsconfig/svelte': '^1.0.0',
+	'@types/compression': '^1.7.0',
+	'@types/node': '^14.0.27',
+	'@types/polka': '^0.5.1',
+	'svelte-check': '^1.0.0',
+	'svelte-preprocess': '^4.0.0',
+	'typescript': '^3.9.3',
```

Comment:
should we use TypeScript 4.0 here?

Reply:
I bumped it to v4, using it without any issues on own project as well.